scripts/pylib/twister/twisterlib.py

```diff
@@ -587,6 +587,10 @@ def handle(self):
                        ] + command
             run_valgrind = True
 
+        # Only valid for native_posix
+        if self.seed is not None:
+            command = command + ["--seed="+str(self.seed)]
```

Comment:
Can we rename the argument to `--test-seed`?

Reply:
This argument is defined in `fake_entropy_native_posix.c`.  If we rename it we may affect others who use this arg as part of their test setup.